CheckManga/schema.sql

```diff
@@ -0,0 +1,34 @@
+DROP TABLE IF EXISTS manga;
+DROP TABLE IF EXISTS sites;
+DROP TABLE IF EXISTS tags;
+DROP TABLE IF EXISTS completed;
+
+CREATE TABLE sites(
+	SiteName TEXT PRIMARY KEY,
+);
+CREATE TABLE manga(
+	Title TEXT PRIMARY KEY,
+	Site TEXT,
+	Status TEXT,
+	Url TEXT,
+	LastChapterRead TEXT,
+	TimeLastUpdated TEXT,
+	TimeLastChecked TEXT,
+	MostRecentChapter TEXT,
+	CoverImageUrl TEXT,
+	FOREIGN KEY(Site) REFERENCES sites(SiteName)
+);
+CREATE TABLE tags(
+	ID INT PRIMARY KEY,
+	Site TEXT,
+	TagName TEXT,
+	TagClass TEXT,
+	FOREIGN KEY(Site) REFERENCES sites(SiteName)
+);
+CREATE TABLE completed(
```

Comment:
given that there is a 1-1 completed to manga relationship, you can prob just combine these two tables

Reply:
That's fair but what should I do if I want the completed titles to have ratings/reviews associated with them whereas the ongoing/bookmarked ones should not?